portfolio/src/main/webapp/js/script.js

```diff
@@ -53,9 +53,39 @@ async function getText() {
   }
 }
 
+/**
+ * Display text from data Servlet
+ */
+async function getText() {
+  var numComments = document.getElementById("number-comments");
+  numComments = numComments.options[numComments.selectedIndex].value;
```

Comment:
this is generally not good practice - you are changing the type of this variable.

Try doing this instead to reduce the number of extra variables too:
```
getNumComments(numComments.options[numComments.selectedIndex].value)`
```